R_Core_Developers.Rmd

```diff
@@ -0,0 +1,34 @@
+# R Core Developers
+
+This page lists the former and current members of the R Core team who have write access to the R source.
+
+  * Brian Ripley (present) 
+  * Deepayan Sarkar (present) 
+  * Douglas Bates (present) 
+  * Duncan Murdoch (up to September 2017)
+  * Duncan Temple Lang (present) 
+  * Friedrich Leisch (present)
+  * Guido Masarotto (up to June 2003)
+  * Heiner Schwarte (up to October 1999)
+  * John Chambers (present)
+  * Kurt Hornik (present)
+  * Luke Tierney (present)
+  * Martin Maechler (present)
+  * Martin Morgan (present)
+  * Martyn Plummer (present)
+  * Michael Lawrence (present)
+  * Paul Murrell (present)
+  * Peter Dalgaard (present)
+  * Robert Gentleman (present)
+  * Ross Ihaka (present)
+  * Seth Falcon (up to August 2015)
+  * Simon Urbanek (present)
+  * Stefano lacus (up to July 2014)
+  * Thomas Lumley (present)
+  * Tomas Kalibera (present)
+  * Uwe Ligges (present)
+
+Affiliations of R Core team members are available [here](). 
+<!-- TODO add URL to bios, perhaps on the R project website -->

```

Comment:
twitter handles where applicable?

Reply:
You should avoid using "here" as hypertext, see e.g. https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text. So you might change this to "View the [affiliations of R Core members]()".